packages/modules/promotion/src/migrations/Migration20240617102917.ts

```diff
@@ -0,0 +1,22 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240617102917 extends Migration {
```

Comment:
suggestion: I dont see the snapshot being updated, would that create issue if we generate another migration?

Reply:
oh got it 💪